api/core/settings.py

```diff
@@ -209,7 +209,7 @@
         'default': {
             'ENGINE': 'django.db.backends.postgresql',
             'NAME': 'voron',
-            'USER': os.environ.get('USER'),
+            'USER': 'postgres',
```

Comment:
revert this please. If you need, you can set USER=postgres in your `.env` file

Reply:
done